lib/src/core/better_player_controller.dart

```diff
@@ -1181,14 +1187,16 @@ class BetterPlayerController {
         !(videoPlayerController?.value.hasError ?? true);
 
     if (isPipSupported && canEnablePictureInPicture) {
-      if (isFullScreen) exitFullScreen();
+      _isPip = true;
+      exitFullScreen();
```

Comment:
hmm tutaj jest exitFullScreen, a niżej kawałek enter full screen - o co chodzi? czemu tak?

Reply:
sprawdź jeszcze bo na andoridzie tak mam ze jak odblokuje rotacje, i wlacze pip to moge rotowac ekranem i wyjdzie mi on z ful screena

Reply:
exitFullScreen musi być uzyta dla Androida bo jesli przechodzi do pip to nie mamy full screen na pip, aktualizacja stanu rzeczy przy pip w BetterPlayerWithControls dzieje sie gdy uruchomiony zostanie jakiś event, zmodyfikowałem to tak żeby działało jak wczesniej dla Androida, dla iOSa ten exitFullScreen jest uruchamiany bo bez tego nie widzi mi flagi iPip i nie pokazuje się overlay. Didałem tam eventy dla pip ale to nadal na nie nie reagowało więc zostawiłem ten exitFullScreen